README.md

```diff
@@ -288,9 +291,29 @@ make test_unit
 make test_e2e_shannon_relay
 ```
 
-## 8. Troubleshooting
+## 8. Running Localnet
 
-### 8.1. Docker Permissions Issues - Need to run sudo?
+You can use path configuration under `/local` to spin up a local development environment using `Kind` + `Tilt`.
```

Comment:
Let's add the same explanation of why we need kind

cc @commoddity 

<img width="1832" alt="Screenshot 2024-12-03 at 7 45 43 PM" src="https://github.com/user-attachments/assets/416fcf98-7c00-497f-a713-2b99281d5795">

Reply:
Added a comment explaining the potential situation when using K8s Docker desktop.